code/controllers/subsystem/transport.dm

```diff
@@ -0,0 +1,160 @@
+PROCESSING_SUBSYSTEM_DEF(transport)
+	name = "Transport"
+	wait = 0.5
```

Comment:
0.5 what? Use the correct interval define please

Reply:
Ah yeah, I just copied the existing value. Fixed.